cfe_internal/enterprise/federation/federation.cf

```diff
@@ -129,6 +129,44 @@ bundle agent config
       "I'm paused so won't do any import/dump";
 }
 
+bundle agent semanage_installed
+# @brief Install semanage utility if selinux enabled and
+# cfengine_mp_fr_dependencies_auto_install class is defined
+# if not defined then only warn
+{
+  vars:
+    "semanage_action"
+      string => ifelse( "default:_stdlib_path_exists_semanage", "fix", "default:cfengine_mp_fr_dependencies_auto_install", "fix", "warn" ),
+      comment => "We only want to use semanage if it's available, or if we have
+                  indicated it's ok to install it automatically. This variable
+                  is subsequently used by a commands and packages promises to
+                  warn or fix based.";
+    debian_6|debian_7|debian_8|ubuntu_12|ubuntu_14|ubuntu_16|rhel_5::
+      "semanage_package" string => "policycoreutils";
+    debian_9|debian_10|ubuntu_18|redhat_8|centos_8::
+      "semanage_package" string => "policycoreutils-python-utils";
+    redhat_6|centos_6|redhat_7|centos_7::
+      "semanage_package" string => "policycoreutils-python";
```

Comment:
It might be more readable in a single promise using ifelse()

```
"semanage_pacakge" string => ifelse( "debian_6|debian_7|debian_8|ubuntu_12|ubuntu_14|ubuntu_16|rhel_5", "policycoreutils",
"redhat_6|centos_6|redhat_7|centos_7", "policycoreutils-python",
"policycoreutils-python-utils"); # Seems like the newest releases all use this package name, so this /might/ keep working for debian 11 and centos 9 whenever they come out.
```

Reply:
awesome! Will make that change in a follow-up!